pkg/messagebroker/kafka.go

```diff
@@ -106,11 +106,13 @@ func newKafkaProducerClient(ctx context.Context, bConfig *BrokerConfig, options
 	configMap := &kafkapkg.ConfigMap{
 		"bootstrap.servers":       strings.Join(bConfig.Brokers, ","),
 		"socket.keepalive.enable": true,
-		"retries":                 2,
-		"linger.ms":               0,
-		"request.timeout.ms":      30000,
-		"delivery.timeout.ms":     150000,
+		"retries":                 3,
```

Comment:
we should remove the internal retires here, any retry behaviour we want to add should be done at messagebroker level

Reply:
These will retry any intermittent network drops. I have reduced the timeout as well. So all the request with retries will complete within 10sec for now.

Reply:
Yes, but this is only in kafka, we will need to have a constant behaviour for all brokers. also this config has issues with ordering. we will need to handle it outside 

https://docs.confluent.io/platform/current/installation/configuration/producer-configs.html#producerconfigs_retries